main.js

```diff
@@ -139,36 +139,25 @@ function startApp() {
     }
   }
 
-  function handleUnhandledException(err) {
-    bt.report(err, {}, handleFinishedReport);
-  }
-
-  if (pjson.env === 'production') {
-    bt.initialize({
-      disableGlobalHandler: true,
-      endpoint: 'https://streamlabs.sp.backtrace.io:6098',
-      token: 'e3f92ff3be69381afe2718f94c56da4644567935cc52dec601cf82b3f52a06ce',
-      attributes: {
-        version: pjson.version,
-        processType: 'main'
-      }
-    });
+  Raven.config('https://6971fa187bb64f58ab29ac514aa0eb3d@sentry.io/251674', {
```

Comment:
Unless I'm missing something, it looks like the production check was removed.  I'm assuming this was for testing, but we should add it back in before merging.l

Reply:
Oh I totally forgot, thanks for pointing, fixed!